src/app/modules/verification/verification.util.ts

```diff
@@ -155,6 +167,11 @@ export const mapRouteError: MapRouteError = (
         errorMessage: `You must wait for ${WAIT_FOR_OTP_SECONDS} seconds between each OTP request.`,
         statusCode: StatusCodes.UNPROCESSABLE_ENTITY,
       }
+    case OtpRequestCountExceededError:
+      return {
+        errorMessage: `You have requested for too many OTPs. Please refresh and try again.`,
```

Comment:
reads a little weird haha, maybe
```suggestion
        errorMessage: `You have requested too many OTPs. Please refresh and try again.`,
```

Reply:
updated in 0f5705b4